src/Context/Node/NodeContext.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @author Sergii Bondarenko, <sb@firstvector.org>
+ */
+namespace Drupal\TqExtension\Context\Node;
+
+use Drupal\TqExtension\Context\RawTqContext;
+
+class NodeContext extends RawTqContext
+{
+    /**
+     * @When /^I (visit|view|edit) "([^"]+)" node of type "([^"]+)"$/
+     */
+    public function iVisitNodePageOfType($operation, $title, $type) {
+        if ($operation == 'visit') {
+            $operation = 'view';
+        }
+        $query = new \EntityFieldQuery();
```

Comment:
LIke can be dangerous as the title we're searching can be anything, even a small string like 'test article' and the DB can already have 'How to create a test article" node for example. I think being strict here is a good thing ;)